spec/supabase_js_v2_legacy.yml

```diff
@@ -375,7 +375,7 @@ pages:
           ```
 
   auth.getUser():
-    title: 'getUser()'
+    title: 'getUserById()'
```

Comment:
@kiwicopple shouldn't the title be `getUser` here?

Reply:
yup